packages/dds/tree/src/feature-libraries/modular-schema/genericFieldKind.ts

```diff
@@ -56,7 +55,10 @@ export const genericChangeHandler: FieldChangeHandler<GenericChangeset> = {
 				markList.push({ count: offset });
 				nodeIndex = index;
 			}
-			markList.push({ count: 1, fields: deltaFromChild(nodeChange) });
+			const childDelta = deltaFromChild(nodeChange);
+			if (childDelta !== undefined) {
+				markList.push({ count: 1, fields: childDelta });
+			}
```

Comment:
This change introduces a bug: omitting the entry altogether you change which index the following entries affect. This change should not be needed.

Reply:
Oh I see, that makes sense, I didn't realize just how finnicky this stuff is